libs/tree-sitter-wing/src/scanner.c

```diff
@@ -169,8 +169,6 @@ static bool scan_automatic_block(TSLexer *lexer)
       return false;
     if (lexer->is_at_included_range_start(lexer))
       return true;
-    if (lexer->lookahead == '\n')
-      break;
```

Comment:
This was copied from the automatic semicolon insertion, but it's not a helpful heuristic here.